tools/eslint-rules/prefer-common-mustnotcall.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileoverview Prefer common.mustNotCall(msg)
```

Comment:
Nit: Indicate what you are preferring it over. Maybe `Prefer common.mustNotCall(msg) over common.mustCall(fn, 0)` or whatever.